pkg/controller/plan/controller.go

```diff
@@ -262,6 +269,25 @@ func (r Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (r
 	return
 }
 
+func (r *Reconciler) setPopulatorDataSourceLabels(plan *api.Plan) {
+	if plan.Status.HasAnyCondition(Succeeded, Failed, Canceled) {
```

Comment:
why do we check this?

Reply:
talked offline, the missing bit here is for new plans - probably to use the mutator to annotate it when a new plan is created. for existing plans it's ok, but it would be better to figure if the plan is new and didn't run to annotate it immediately.

Reply:
filed https://github.com/kubev2v/forklift/issues/366